dcpy/models/lifecycle/ingest.py

```diff
@@ -45,20 +45,24 @@ class PreprocessingStep(BaseModel):
     mode: str | None = None
 
 
+class DatasetAttributes(BaseModel):
+    name: str | None = None
+    description: str | None = None
+    url: str | None = None
+    custom: dict | None = None
+
+
 class Template(BaseModel, extra="forbid", arbitrary_types_allowed=True):
     """Definition of a dataset for ingestion/processing/archiving in edm-recipes"""
 
     id: str
     acl: recipes.ValidAclValues
 
-    target_crs: str | None = None
+    attributes: DatasetAttributes | None = None
```

Comment:
maybe make it a required attribute? To avoid creation of new templates without relevant dataset info

Reply:
That seems like the right choice